app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "23.0.2"
     defaultConfig {
         applicationId "com.zulip.android"
-        minSdkVersion 13
+        minSdkVersion 16
```

Comment:
This is a change that potentially has a pretty big impact on the Zulip install base. What's the reason for upping this to 16?